### ISSUE TITLE
Thunderbird add-on

### ISSUE THREAD

**Issue by Anaud** [39aldo39/DecSync#1]
Hi, is a Thunderbird version planned for the future?
Great job by the way!

**Comment by 39aldo39** [comment]
No, there is currently no Thunderbird version planned. Someone else could develop one, but currently you can use the Evolution version instead. Another possibility is to develop a local WebDAV/CalDAV server with DecSync as backend, which adds support for most programs.
